ui/framework/src/test/frontstage/FrontstageDef.test.tsx

```diff
@@ -187,15 +187,15 @@ describe("float and dock widget", () => {
     sinon.stub(UiFramework, "uiVersion").get(() => "2");
     sinon.stub(frontstageDef, "nineZoneState").get(() => state).set(nineZoneStateSetter);
     sinon.stub(frontstageDef, "nineZoneState").get(() => state);
-    frontstageDef.floatWidget("t1", {x:55, y:105});
+    frontstageDef.floatWidget("t1", { x: 55, y: 105 });
     nineZoneStateSetter.calledOnce.should.true;
 
     frontstageDef.floatWidget("ta");
     nineZoneStateSetter.calledOnce.should.true;
   });
 
   it("should not float if v1", () => {
-    let state = createNineZoneState({size: {height: 1000, width: 1600}});

```

Comment:
@calebmshafer - Do you have some extension loaded that adds spaces?  If not, why would eslint add them for you and not for us? The reason I ask is sometimes I feel VS Code is not consistently adding spaces on file save and I am not sure why.

Reply:
Possibly an Extension is missing from our recommended settings. I'll take a pass now but I do have 20+ extensions installed :)

Do you have all recommended Extensions installed?  `.vscode/extensions.json`

Reply:
Yep, Got all the recommended ones. It does seem to format spaces properly at least 90% of time, so I should be happy about that.

Reply:
Sounds good. I looked through my VSCode settings and I don't see anything that should effect this. The only difference most likely is I'm running on Mac vs. Windows.  I'll see if it auto-formats for me on Windows in this case.